libs/openchallenges/home/src/lib/_home-theme.scss

```diff
@@ -0,0 +1,31 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+@mixin color($theme) {
+  $config: mat.get-color-config($theme);
+  $primary: map.get($config, 'primary');
+  $accent: map.get($config, 'accent');
+  $warn: map.get($config, 'warn');
+  $figma: map.get($config, 'figma');
+
+  #announcement {
+    background-color: rgba(#ede8ff, 0.4);
+    border-left-color: #8a78ff;
```

Comment:
Could we use a brighter and/or contrasting color?

The purple makes the banner look part of the home page.